pep-0503.txt

```diff
@@ -54,12 +54,12 @@ a repository. The format of this URL is ``/<project>/`` where the ``<project>``
 is replaced by the normalized name for that project, so a project named
 "HolyGrail" would have an URL like ``/holygrail/``. This URL must respond with
 a valid HTML5 page with a single anchor element per file for the project. The
-text of the anchor tag **MUST** be the filename of the file and the href
-attribute **MUST** be an URL that links to the location of the file for
-download. The URL **SHOULD** include a hash in the form of an URL fragment with
-the following syntax: ``#<hashname>=<hashvalue>``, where ``<hashname>`` is the
-lowercase name of the hash function (such as ``sha256``) and ``<hashvalue>`` is
-the hex encoded digest.
+href attribute **MUST** be an URL that links to the location of the file for
```

Comment:
Maybe just "links to the file..." rather than "links to the location of the file..." - the latter could be read as allowing a link to the directory in which the file resides.

Reply:
That’s from the original text, so fw @dstufft

Reply:
The wording of the second half of the revised sentence makes it clear that the last part of the URL has to be the filename itself, not the directory containing the file.